core/cachestub/batch_cache_stub.go

```diff
@@ -0,0 +1,57 @@
+package cachestub
+
+import (
+	"github.com/anoideaopen/foundation/proto"
+	"github.com/hyperledger/fabric-chaincode-go/shim"
+)
+
+type BatchCacheStub struct {
+	shim.ChaincodeStubInterface
+	batchCache map[string]*proto.WriteElement
```

Comment:
Please rename to batchWriteCache

Reply:
Of course, done

Reply:
You renamed the wrong thing.
The stub should be left as it was, but the field inside the structure should be renamed